lib/instance-validator.js

```diff
@@ -351,24 +369,40 @@ class InstanceValidator {
   _handleReflectedResult(field, value, promiseInspections) {
     for (const promiseInspection of promiseInspections) {
       if (promiseInspection.isRejected()) {
-        const rejection = promiseInspection.error();
-        this._pushError(true, field, rejection, value);
+        const
+          rejection = promiseInspection.error(),
+          isBuiltIn = !!rejection.validatorName;
```

Comment:
aha I see ... also the argument for minimizing diffs is a really good one! (and I'm glad I'm not the only OCD sufferer that is annoyed that "let" and "const" have different string lengths!)